### PR TITLE
Дз 5

### PR DESCRIPTION
**Pull request by 5OR627**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -91,4 +91,7 @@
         else:
             loss += profit
```
Comment (ShapovaLOVALOVA) [inline main.py:92]: переменная loss получается накопительная сумма потерь всех фирм
